src/compressed_tensors/quantization/quant_scheme.py

```diff
@@ -110,15 +111,88 @@ def is_preset_scheme(name: str) -> bool:
     return name.upper() in PRESET_SCHEMES
 
 
-W8A8 = dict(weights=QuantizationArgs(), input_activations=QuantizationArgs())
-
-W4A16 = dict(weights=QuantizationArgs(num_bits=4, group_size=128))
+# 8 bit weights only quantization
+W8A16 = dict(
+    weights=QuantizationArgs(
+        num_bits=8,
+        type=QuantizationType.INT.value,
+        strategy=QuantizationStrategy.CHANNEL.value,
+        symmetric=True,
+        dynamic=False,
+    ),
+)
+# 4 bit weights only quantization
+W4A16 = dict(
+    weights=QuantizationArgs(
+        num_bits=4,
+        type=QuantizationType.INT.value,
+        group_size=128,
+        strategy=QuantizationStrategy.GROUP.value,
+        symmetric=True,
+        dynamic=False,
+    ),
+)
+# 8 bit weights and 8 bit activations quantization
+W8A8 = dict(
+    weights=QuantizationArgs(
+        num_bits=8,
+        type=QuantizationType.INT.value,
+        strategy=QuantizationStrategy.CHANNEL.value,
+        symmetric=True,
+        dynamic=False,
+    ),
+    input_activations=QuantizationArgs(
+        num_bits=8,
+        type=QuantizationType.INT.value,
+        strategy=QuantizationStrategy.TENSOR.value,
+        symmetric=True,
+        dynamic=True,
+    ),
+)
+# 4 bit weights and 8 bit activations quantization
+W4A8 = dict(
+    weights=QuantizationArgs(
+        num_bits=4,
+        type=QuantizationType.INT.value,
+        group_size=128,
+        strategy=QuantizationStrategy.GROUP.value,
+        symmetric=True,
+        dynamic=False,
+    ),
+    input_activations=QuantizationArgs(
+        num_bits=8,
+        type=QuantizationType.INT.value,
+        strategy=QuantizationStrategy.TENSOR.value,
```

Comment:
Ditto, although I don't know what we should target here as we don't really have a performant implementation yet, aside from QQQ

Reply:
I agree. INT8 activation quantization should be per token